src/io/flutter/logging/FlutterLogView.java

```diff
@@ -344,6 +346,31 @@ private String countString(int count) {
     }
   }
 
+  private static class LogVerticalScrollChangeListener implements ChangeListener {
+    private volatile int oldScrollValue = 0;
+
+    @Override
+    public void stateChanged(ChangeEvent e) {
+      final BoundedRangeModel model = (BoundedRangeModel)e.getSource();
```

Comment:
I'm not too familiar w/ `ChangeEvent`s but should we do an instance check to be safe or is this cast a sure thing?

Reply:
e.getSource() should be `BoundedRangeModel`.
But for sure, I just add check `instanceof BoundedRangeModel`